app.nestful.Nestful.yml

```diff
@@ -0,0 +1,35 @@
+id: app.nestful.Nestful
+
+runtime: org.gnome.Platform
+runtime-version: '47'
+sdk: org.gnome.Sdk
+tags:
+  - proprietary
+command: nestful
+finish-args:
+  - --socket=wayland # Permission needed to show the window
+  - --socket=fallback-x11 # Permission needed to show the window
+  - --device=dri # OpenGL, not necessary for all projects
+  - --share=ipc
+
+modules:
+  - name: binary
+    buildsystem: simple
+    sources:
+      - type: file
+        path: ./app.nestful.Nestful.metainfo.xml
+      - type: file
+        dest-filename: Nestful.deb
+        url: https://cdn.crabnebula.app/download/nestful/nestful/latest/platform/debian-x86_64
+        sha256: aea487c12f2fdf85d22b9fea80f36aa237d3e5897f8301f9d7b7152c5e9e87aa
+        only-arches: [x86_64]
```

Comment:
so what happen on aarch64?

Reply:
but you still build an aarch64 package.

Reply:
https://docs.flathub.org/docs/for-app-authors/requirements#flathubjson